charts/timescaledb-single/templates/statefulset-timescaledb.yaml

```diff
@@ -404,7 +415,7 @@ spec:
         image: "{{ .Values.prometheus.image.repository }}:{{ .Values.prometheus.image.tag }}"
         imagePullPolicy: {{ .Values.prometheus.image.pullPolicy }}
         securityContext:
-          runAsUser: 1000
+          allowPrivilegeEscalation: false
```

Comment:
the `runAsUser` has moved to the Pod SecurityContext, and therefore is still set to 1000